device/core/test/_module_client_test.js

```diff
@@ -152,4 +183,117 @@ describe('ModuleClient', function() {
       });
     });
   });
+
+  ['sendOutputEvent', 'sendOutputEventBatch'].forEach(function(funcName) {
+    describe('#' + funcName, function() {
+      /*Tests_SRS_NODE_INTERNAL_CLIENT_18_019: [The `sendOutputEvent` method shall not throw if the `callback` is not passed. ]*/
+      /*Tests_SRS_NODE_INTERNAL_CLIENT_18_022: [The `sendOutputEventBatch` method shall not throw if the `callback` is not passed. ]*/
+      it('doesn\'t throw if no callback is given and the method exists on the transport', function() {
+        var transport = new FakeTransport();
+        var client = new ModuleClient(transport);
+        client.open(function() {
+          assert.doesNotThrow(function() {
+            client[funcName]('message');
+          });
+        });
+      });
+    });
+  });
+
+  describe('#on(\'inputMessage\')', function () {
+    /*Tests_SRS_NODE_INTERNAL_CLIENT_18_012: [ The `inputMessage` event shall be emitted when an inputMessage is received from the IoT Hub service. ]*/
+    it('emits a message event when a message is received', function (done) {
+      var dummyTransport = new FakeTransport();
```

Comment:
>dummyTransport [](start = 10, length = 14)

The PC police say that this is an offensive name.

Reply:
Wait, that's my dev ID.  Was I the person who was being insensitive?

---
In reply to: [195808859](https://github.com/Azure/azure-iot-sdk-node/pull/316#discussion_r195808859) [](ancestors = 195808859)